tests/app/dao/test_services_dao.py

```diff
@@ -783,3 +785,11 @@ def test_fetch_monthly_historical_template_stats_for_service_separates_templates
     assert len(result.get('2016-04').keys()) == 2
     assert str(template_one.id) in result.get('2016-04').keys()
     assert str(template_two.id) in result.get('2016-04').keys()
+
+
+def test_dao_fetch_active_users_for_service_returns_active_only(sample_service):
```

Comment:
perhaps change sample_service to create_service to explicitly show that we are creating 1 active user as well as the pending user